File: src/main/java/ru/siaw/free/rpplugin/utility/Print.java
package ru.siaw.free.rpplugin.utility;

import org.bukkit.Bukkit;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

import java.util.logging.Level;
import java.util.logging.Logger;

public class Print {
    private static final Logger logger = Bukkit.getLogger();

    public static void toConsole(String msg) {
        logger.log(Level.WARNING, () -> msg);
    }

    public static void toPlayer(Player p, String msg) {
        p.sendMessage(msg);
    }

    public static void toSender(CommandSender sender, String msg) {
        sender.sendMessage(msg);
    }

    public static void toPlayers(String msg) {
        Bukkit.getOnlinePlayers().forEach(p -> toPlayer(p, msg));
    }
}

File: src/main/java/ru/siaw/free/rpplugin/utility/Print.kt
package ru.siaw.free.rpplugin.utility

import org.bukkit.entity.Player
import org.bukkit.Bukkit
import org.bukkit.ChatColor

object Print {
    @JvmStatic
    fun msgToConsole(msg: String) {
        println(ChatColor.GREEN.toString() + msg)
    }

    @JvmStatic
    fun infoToConsole(msg: String) {
        println(ChatColor.YELLOW.toString() + msg)
    }

    @JvmStatic
    fun errToConsole(msg: String) {
        println(ChatColor.RED.toString() + msg)
    }

    fun toPlayer(p: Player, msg: String?) {
        p.sendMessage(msg)
    }

    fun toPlayers(msg: String?) {
        for (p in Bukkit.getOnlinePlayers()) toPlayer(p, msg)
    }
}